erpnext/hr/doctype/training_program/training_program.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2017, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Training Program', {
+	refresh: function(frm) {
+		if(frm.doc.status == 'Scheduled') {
```

Comment:
Never hardcode status.

You should also be able to see training events linked to this. Try adding a `dashboard` its pretty simple. (See `material_request_dashboard.py`

Reply:
I have already added a dashboard @rmehta